nav-kibana-dev.docnav.json

```diff
@@ -165,6 +165,48 @@
         { "id": "kibVisTypeTimeseriesPluginApi" },
         { "id": "kibVisualizationsPluginApi" }
       ]
+    },
+    {
+      "label": "Operations",
+      "items": [
+        { "id": "kibDevDocsOpsOverview", "label": "Overview" },
+        {
+          "label": "CI",
+          "items": [
+            { "id": "kibDevDocsOpsCiStats" }
+          ]
+        },
+        {
+          "label": "Toolchain",
```

Comment:
@spalger I've added this new section in between my latest commits as I think it could help us differentiate better. If you prefer to avoid it we can just  merge them together with `build tooling`

Reply:
Idk, I would probably put all these in "build tooling", I want to separate things that are clearly different, but if we create too many groups it will be tricky to tell where the thing you're looking for is.

Reply:
In my head a toolchain is something that touches with compiling, linking & debug code and can be used in a development task where the build tools are pieces of software that could use a toolchain in the effort of shipping software. However it probably doesn't justify the distinction for now. I will regroup them together again on `build tooling` only.

Reply:
Yeah, to me toolchain is super abstract and just indicates some collection of tools which is why I don't understand the distinction.